libs/langchain/langchain/chat_models/vertexai.py

```diff
@@ -122,6 +122,8 @@ class ChatVertexAI(_VertexAICommon, BaseChatModel):
     "Underlying model name."
     examples: Optional[List[BaseMessage]] = None
 
+    tuned_model_name: Optional[str] = None
```

Comment:
is there any way to identify tuned_model_name from the name alone? wondering if we need separate params for model_name vs tuned_model_name

Reply:
cc @lkuligin

Reply:
I guess you could check if the "model_name" is in this list of tuned models:

Reply:
Hello when I try to use the  tuned_model_name parameter, it is ignored. When I look at the latest langchain source I see ChatVertexAI actually doesn't use this at all.
Was this change not merged in at all?
How do I use a tuned model with ChatVertexAI please?